i18n/en.json

```diff
@@ -536,6 +536,38 @@
     "translation": "echo"
   },
   {
+    "id": "api.command_channel_header.desc",
+    "translation": "Edit the channel header."
+  },
+  {
+    "id": "api.command_channel_header.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_header.name",
+    "translation": "header"
+  },
+  {
+    "id": "api.command_channel_header.message.app_error",
+    "translation": "A message must be provided with the /header command."
```

Comment:
Is this necessary anymore? Since we now open the 'set header' dialog if text isn't provided.